### PR TITLE
install: cleanup setup wizard

### PR DESCRIPTION
**Pull request by dcramer**
Refs GH-5699

### CODE REVIEW

File: src/sentry/static/sentry/app/views/installWizard.jsx [hunk 1]
```diff
@@ -87,8 +81,7 @@ const InstallWizardSettings = React.createClass({
     return (
       <form onSubmit={this.onSubmit}>
         <p>
-          Welcome to Sentry, yo! Complete setup by filling out the required
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/installWizard.jsx]: about time we killed this /cc @mattrobenolt

Comment (mattrobenolt) [inline src/sentry/static/sentry/app/views/installWizard.jsx]: lol /cc @ckj

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/installWizard.jsx [hunk 2]
```diff
@@ -87,8 +81,7 @@ const InstallWizardSettings = React.createClass({
...
-          configuration.
+          Complete setup by filling out the required configuration.
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/installWizard.jsx]: translate?

**Pull request closed without merging**